sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/pubsublite/internal/PublisherCache.java

```diff
@@ -27,9 +27,13 @@
 import java.util.HashMap;
 import org.apache.beam.sdk.io.gcp.pubsublite.PublisherOptions;
 import org.apache.beam.vendor.guava.v26_0_jre.com.google.common.annotations.VisibleForTesting;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 /** A map of working publishers by PublisherOptions. */
 class PublisherCache implements AutoCloseable {
+  private final Logger logger = LoggerFactory.getLogger(PublisherCache.class);
```

Comment:
nit: we usually make loggers static and call them LOG:

```suggestion
  private static final Logger LOG = LoggerFactory.getLogger(PublisherCache.class);
```

```
❯ grep -iIrn LoggerFactory\.getLogger sdks/java | grep private | cut -d' ' -f2- |  cut -d = -f-1 | awk '{$1=$1};1' | sort | uniq -c | sort -n
      1 private final Logger log
      1 private static final org.slf4j.Logger LOG
    263 private static final Logger LOG
```